app/helpers/application_helper.rb

```diff
@@ -58,10 +58,8 @@ def project_icon(project_id, options = {})
         Project.find_with_namespace(project_id)
       end
 
-    if project.avatar.present?
-      image_tag project.avatar.url, options
-    elsif project.avatar_in_git
```

Comment:
Looks like you're not taking `avatar_in_git` in account anymore. Is this on purpose?